lib/pages/chat_search/chat_search_view.dart

```diff
@@ -85,7 +85,8 @@ class _ServerSearchView extends StatelessWidget {
         ValueListenableBuilder(
           valueListenable: serverSearchController.searchResultsNotifier,
           builder: (context, searchResults, child) {
-            final events = searchResults.searchResults
+            final events = (searchResults as PresentationServerSideSearch)
```

Comment:
IMO it's safer to make a condition before casting 
`if (searchResults is PresentationServerSideSearch) return child!`

Reply:
Yes, apply it in last commit